databrowser/src/components/header/AppHeader.vue

```diff
@@ -2,15 +2,17 @@
   <header>
     <router-link to="/" class="text-2xl">OPENDATAHUB!</router-link>
     <LanguageSwitcher></LanguageSwitcher>
+    <user-authentication />
```

Comment:
```suggestion
    <UserAuthentication />
```

I would suggest to use this naming convention, because it is closer/the same as the component name.